examples/advanced/main.tf

```diff
@@ -15,16 +15,18 @@
  */
 
 module "cloud-nat" {
-  source                           = "../../"
-  project_id                       = var.project_id
-  region                           = var.region
-  router                           = var.router_name
-  name                             = "my-cloud-nat-${var.router_name}"
-  nat_ips                          = var.nat_addresses
-  min_ports_per_vm                 = "128"
-  icmp_idle_timeout_sec            = "15"
-  tcp_established_idle_timeout_sec = "600"
-  tcp_transitory_idle_timeout_sec  = "15"
-  udp_idle_timeout_sec             = "15"
+  source                             = "../../"
+  project_id                         = var.project_id
+  region                             = var.region
+  router                             = var.router_name
+  name                               = "my-cloud-nat-${var.router_name}"
+  nat_ips                            = var.nat_addresses
+  min_ports_per_vm                   = "128"
+  icmp_idle_timeout_sec              = "15"
+  tcp_established_idle_timeout_sec   = "600"
+  tcp_transitory_idle_timeout_sec    = "15"
+  udp_idle_timeout_sec               = "15"
+  source_subnetwork_ip_ranges_to_nat = var.source_subnetwork_ip_ranges_to_nat
```

Comment:
Examples should ideally require minimal external config.

We should directly incorporate the subnetwork setup into the example.

Reply:
How do you see it?